extensions/u301-url-shortener/package.json

```diff
@@ -11,7 +11,7 @@
   "license": "MIT",
   "commands": [
     {
-      "name": "shorten-clipboard",
+      "name": "index",
```

Comment:
is this correct?

Reply:
Yes, since users might have an alias or hotkey assigned and it's bound by the `name` field